test/Microsoft.AspNet.Mvc.Razor.Test/Compilation/ExpressionRewriterTest.cs

```diff
@@ -456,10 +457,20 @@ public IEnumerable<FieldDeclarationSyntax> FindFields(SyntaxNode node)
 
         private CSharpCompilation Compile(SyntaxTree tree)
         {
+            // Disable 1702 until roslyn turns this off by default
+            var options = new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary)
```

Comment:
Copied from https://github.com/aspnet/dnx/blob/167fff5c19a0864dcf4b31edaf7c17d13bd35a3f/src/Microsoft.Dnx.Compilation.CSharp.Common/CompilerOptionsExtensions.cs#L30-L34 since Roslyn prints assembly redirect warnings in CoreCLR.